src/components/movies/MovieListCard.vue

```diff
@@ -0,0 +1,96 @@
+<script>
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  props: {
+    movieData: {
+      type: Object,
+      required: true,
+    },
+  },
+  computed: {
+    formattedMovieLength() {
+      if (!this.movieData) return '0h 0 min';
+      const allMinutes = this.movieData.length;
+      const hours = Math.floor(allMinutes / 60);
+      const minutes = allMinutes - hours * 60;
+      if (minutes === 0) return `${hours}h`;
+      if (minutes < 10) return `${hours}h 0${minutes} min`;
+      return `${hours}h ${minutes} min`;
```

Comment:
You could utilize `String.padStart()` function here 👀
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart
For example:
```suggestion
      return `${hours}h ${minutes.toString().padStart(2, '0')} min`;
```

Reply:
Amazing! Thank you for that 😄